src/shared/openssl-util.c

```diff
@@ -3,6 +3,7 @@
 #include "openssl-util.h"
 #include "alloc-util.h"
 #include "hexdecoct.h"
+#include "random-util.h"
```

Comment:
hmm, for some reason the list of includes is not alphabetically sorted (pre-existing issue, unrelated to your PR). But if you touch this part, please fix this at the same time, i.e. make sure that the includes are ordered alphabetically, as our coding style says